### PR TITLE
added some verbiage to the vocabularies example for clarity

### PR DESCRIPTION
**Pull request by gregsdennis**
Some of the text in the example isn't proper "spec-ese" but in the context of an example, I think that's fine.

### CODE REVIEW

File: jsonschema-core.xml [hunk 1]
```diff
@@ -1121,6 +1143,31 @@
...
+                    declares this meta-schema for their "$schema".  Note that defining
+                    it in this way does not imply the new keywords are valid within an
+                    instance to be validated by such a schema.
+                </t>
+                <t>
+                    Although the meta-schema can validate the syntax of "startDate"
+                    within a schema, it is the vocabulary that defines the logic behind
+                    the semantic meaning of "startDate".  Without an understanding of
+                    the semantics (that the instance value should represent a date after that
+                    given in the schema), an implementation can only validate the syntactic
```
Comment (Relequestual) [inline jsonschema-core.xml]: > that the instance value should represent a date after that given in the schema

Would it be valid to add this in the comment for this field, showing that information is contained in annotation which isn't avilable in the meta-schema validation mode, which is required for "proper use" of the vocabulary?

File: jsonschema-core.xml [hunk 2]
```diff
@@ -1121,6 +1143,31 @@
...
+                    usage of "startDate" (that it is a date-formatted string).  This is why
+                    a schema MUST refuse to validate a schema which uses a meta-schema
```
Comment (Relequestual) [inline jsonschema-core.xml]: If this is purely informative and not normative (as in it's already covered earlier), then lowercase that `MUST`, imho.

**Comment by Relequestual** [comment]
"Request changes" is more for me =]

File: jsonschema-core.xml [hunk 3]
```diff
@@ -935,8 +935,9 @@
                     such schemas.
                 </t>
                 <t>
-                    When processing a schema that uses unrecognized vocabularies, keywords
-                    declared by those vocabularies are treated like any other unrecognized
+                    When processing a schema that uses an unrecognized vocabulary
+                    (implying that the value for the vocabulary must be false), keywords
```
[the remark this answers is not in the document]
Comment (gregsdennis) [reply jsonschema-core.xml]: Sure. Have a go.

File: jsonschema-core.xml (hunk 1 above)
Comment (gregsdennis) [reply jsonschema-core.xml]: Good catch. I changed the example mid-way through. I need to update this note.

Comment (gregsdennis) [reply jsonschema-core.xml]: Possibly, sure. But I don't know that it's necessary for this example.  I put it here to contrast the syntactic usage later in the sentence.

File: jsonschema-core.xml (hunk 2 above)
Comment (gregsdennis) [reply jsonschema-core.xml]: ✔️

**Comment by Relequestual** [comment]
Looks like I'm using an older version of xml2rfc.
We should make sure the required version is installed when running make.

**Comment by gregsdennis** [comment]
This looks good to me.

**Comment by gregsdennis** [comment]
Seems like @handrews went his own way.  Probably don't need this in the spec anymore.

Might still be useful for the site, though.

**Comment by handrews** [comment]
@gregsdennis I had actually not intended to invalidate this and had meant to come back and look at reconciling, but perhaps it's better for me to just finish the PR I have up and we can see what's still needed.

**Pull request closed without merging**